### PR TITLE
Dynamic page routing

### PR DESCRIPTION
**Pull request by praveenrach**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-2461

### Change description ###
DA CA Reasonable Adjustments - Navigation from the Disability Start screen 1 DEV

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app/case/definition.ts [hunk 1]
```diff
@@ -2388,6 +2388,24 @@ export interface WithoutNoticeOrderDetails {
   orderWithoutGivingNotice?: YesOrNo;
 }
 
+export enum ApplicantReasonableAdjustments {
```
Comment (khurshedk) [inline src/main/app/case/definition.ts:2391]: why these two enums,can't we use one enum  here

File: src/main/app/case/definition.ts [hunk 2]
```diff
@@ -2622,4 +2640,4 @@ export interface C1ASafteyConcerns {
       firstName: string;
       lastName: string;
       partyType: PartyType;
-  }
```
Comment (khurshedk) [inline src/main/app/case/definition.ts:2625]: remove unnecessary space

File: src/main/steps/applicant/support-you-need-during-case/summary/content.test.ts [hunk 3]
```diff
@@ -11,24 +11,12 @@ const en = {
     aboutYou: 'About you',
   },
   keys: {
-    communicationSupportOther: 'Describe what you need',
-    courtComfort: 'I need something to make me feel comfortable during a court hearing',
-    courtHearing: 'I would need to bring support with me to a court hearing',
-    courtProceedingProvideDetails: 'Provie details',
-    describeOtherNeed: 'Describe what you need',
```
Comment (khurshedk) [inline src/main/steps/applicant/support-you-need-during-case/summary/content.test.ts:18]: wy these fields removed, do we not need them?

Comment (swaroopa86) [reply src/main/steps/applicant/support-you-need-during-case/summary/content.test.ts:18]: Summary screen should be displayed based on the routing, so we have added additional conditions to it

File: src/main/steps/respondent/support-you-need-during-case/summary/content.test.ts [hunk 4]
```diff
@@ -19,25 +19,6 @@ const en = {
...
     respondentReasonableAdjustments:
       'Do you have a physical, mental or learning disability or health condition that means you need support during your case?',
-    respondentDocsSupport: 'I need documents in an alternative format',
-    respondentDocsDetails: 'Please provide the docs details',
-    respondentLargePrintDetails: 'Please provide the large print details',
-    respondentOtherDetails: 'Please provide the other details',
-    respondentHelpCommunication: 'I need help communicating and understanding',
-    respondentSignLanguageDetails: 'Please provide sign language details',
-    respondentDescribeOtherNeed: 'Please provide the details',
-    respondentCourtHearing: 'I would need to bring support with me to a court hearing',
```
[the remark this answers is not in the document]
Comment (swaroopa86) [reply src/main/steps/respondent/support-you-need-during-case/summary/content.test.ts:29]: Summary screen should be displayed based on the routing, so we have added additional conditions to it

**Pull request merged**